plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/AssignAnalysis.java

```diff
@@ -76,7 +76,7 @@ public DefinitionAndCompletions getAssignCompletions(ICodeCompletionASTManager m
                 SourceModule sourceModule = (SourceModule) module;
 
                 try {
-                    defs = sourceModule.findDefinition(state, state.getLine() + 1, state.getCol() + 1,
+                    defs = sourceModule.findDefinition(state, state.getLine(), state.getCol(),
```

Comment:
Humm, this is weird... the docs say that `state.getLine()` returns an int starting at 0 and sourceModule.findDefinition says it expects an int starting at 1, so, this change seems a bit odd for me (i.e.: from the docs the previous code was right).

The docs may be misleading (in which case they should be fixed), but it's also possible that this fix is not correct (if for instance the issue is that the test is not passing the proper index or the state that arrives here wasn't set to have the 1 or 0 based index set correctly).

Unfortunately historically the AST is 1-based while many other things are 0-based (so, timewise it's not worth to change everything to 0-based ints -- one thing I planned to do in the future is creating a special type to say whether the index is 0 or 1 based -- but that's probably a huge change timewise too).

So, please review that, either updating the docs and checking for occurrences which may be wrong -- you can find those with `Ctrl+Shift+G` or this needs to be reverted or reverting this change (and if needed update the code at a different place so that the index arrives properly here).